app/service/rest_svc.py

```diff
@@ -227,7 +227,8 @@ async def add_manual_command(self, access, data):
             return dict(error='Agent missing specified executor')
 
         encoded_command = self.encode_string(data['command'])
-        ability = Ability(ability_id='auto-generated', tactic='auto-generated', technique_id='auto-generated',
+        ability_id = str(uuid.uuid4())
+        ability = Ability(ability_id=ability_id, tactic='auto-generated', technique_id='auto-generated',
```

Comment:
can combine extraneous variable and just set ability_id in the ability instantiation